### ISSUE TITLE
Issue with engrampa 

### ISSUE THREAD

**Issue by Tus1688** [netblue30/firejail#4346]
- Describe the bug.
I can't open, extract file using engrampa
- What did you expect to happen?
successfully extract/open using engrampa 

- What did you do to make it work?
I deleted the symlink using `sudo rm /usr/local/bin/engrampa`, finally I can use it as it supposed to be. 

- Steps to reproduce the behavior:
1. I went to caja file manager
2. I double click .zip file, then engrampa crashed (it doesn't show anything and there is a prompt to wait / force quit).
3. right click -> extract here, then engrampa also crashed .
4. right click -> extract to, then engrampa also crashed.
5. `firejail --list` result = 
```
3545:user::/usr/bin/firejail /usr/bin/engrampa --default-dir=file:///home/user/Downloads --extract file:///home/user/Downloads/blabla.zip 
```
6. I open the zip file from firefox download history, it works 
7. I delete engrampa symlink using `sudo rm /usr/local/bin/engrampa`
8. I did step 2-4 and it works.

 - Linux distribution and version
 Ubuntu 20.04.2 LTS
 - Firejail version 
 0.9.64.4

Thanks in advance for your kindly help :)

**Comment by rusty-snake** [comment]
There were two commits/PRs with fixes for engrampa (#4094 and #4102). Likely engrampa will work when again if you apply them to your engrampa.profile:

```diff
-dbus-user none
+dbus-user filter
+dbus-user.talk ca.desrt.dconf
-memory-deny-write-execute
```

**Comment by Tus1688** [comment]
Thanks ! +1:
